Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# v1.16.4
```

Comment:
I'd like to be conservative and bump to 1.17.0 for more flexibility

Reply:
What does "more flexibility" mean exactly? The downside of bumping is that this is the end of patch releases that can be trivially picked up by the C++ SDK that builds on top of this.

Reply:
I should clarify: C++ releases are going to be more infrequent than iOS ones so I'd like preserve the flexibility to release fixes there to those users.

Meanwhile, this change is functionally transparent to iOS users. It's not an addition or a removal--and it doesn't matter what our intent is.

Reply:
OK, it should be fine to leave as a patch for Firestore.  I feel more strongly about pods with intra-repo dependencies for flexibility in being able to patch if something does go wrong and minimizing the version exponential surface area with different file structures.